src/ape_cache/models.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String  # type: ignore
+
+from .base import Base
+
+
+class Blocks(Base):
```

Comment:
Is there an easier way to generate the table models from core API models

Reply:
No, not really. We need to define the keys of the table. https://github.com/tiangolo/pydantic-sqlalchemy -> Sebastian Ramirez has this issue as well, but this is experimental.

This is a big time spike project I believe. I cannot find anything quickly here.